Source/ParameterEncoding.swift

```diff
@@ -236,7 +236,7 @@ public enum ParameterEncoding {
             while index != string.endIndex {
                 let startIndex = index
                 let endIndex = index.advancedBy(batchSize, limit: string.endIndex)
-                let range = Range(start: startIndex, end: endIndex)
+                let range = Range(startIndex ..< endIndex)
```

Comment:
Wouldn't `startIndex ..< endIndex` return a range? Passing it down to the Range initialiser shouldn't be necessary